Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
```

Comment:
I dont think we need this because at line 8 we are copying entire directory. (see below)

Reply:
agreed, will test without this line and commit if everything  works.

Reply:
actually the line #8 helps to install requirements first, before we copy our source code. 
In case any issues while installing dependencies, docker will fail fast. Though we may not have many source files, its always recommended to install dependencies first and then copy sources.
I just followed nodejs build process. Currently I copied current directory but later someone can maintain their own source directory and copy only that dir.

Please let me know if this works, we can always remove this line if not required